MatrixKit/Views/MXKReceiptSendersContainer.h

```diff
@@ -54,6 +54,11 @@ typedef NS_ENUM(NSInteger, ReadReceiptsAlignment)
  */
 @property (nonatomic) MXRestClient* restClient;
 
+/*
+ The MXSession used to format the events in the details view
+ */
+@property (nonatomic) MXSession *mxSession;
```

Comment:
We don't need the matrix session here. The receipt description will be created in the RR details view controller.